qa/common/util.sh

```diff
@@ -66,7 +66,7 @@ function wait_for_server_ready() {
 
     local wait_secs=$wait_time_secs
     until test $wait_secs -eq 0 ; do
-        if ! kill -0 $spid; then
+        if ! kill -0 $spid > /dev/null 2>&1; then
```

Comment:
Note these changes in `qa/common/util.sh` are to mute log spam around calling `kill` on server pid when it doesn't start successfully.